### ISSUE TITLE
Old, already read chapters appearing in update tab and downloading

### ISSUE THREAD

**Issue by DatHypnoboi** [tachiyomiorg/tachiyomi-extensions#10185]
### Source information

Flame Scans 1.2.11

### Steps to reproduce

1. Add certain Flame Scans series to library, such as Omniscient Reader’s Viewpoint
2. Mark all chapters as read
3. Update library

### Expected behavior

No chapters should appear in the updates tab from Omniscient Reader’s Viewpoint, because there are no new chapters, and old chapters should not start downloading

### Actual behavior

Old chapters appear in the updates tab and begin downloading

### Tachiyomi version

TachiyomiSY 1.7.0

### Android version

Android 12

### Other details

This issue is taking place on TachiyomiSY instead of vanilla Tachiyomi. I also noticed this happen to Trapped in a Webnovel as a Good-for-Nothing from Alpha Scans, though only once. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
This happens because they change their URLs a lot. Enable permanent URL in the extension settings and it should fix this for a while.
